src/board.py

```diff
@@ -84,6 +84,7 @@ def move_piece(self, piece, die_roll):
             piece_to_take.location = self.__taken_location(piece_to_take.colour)
 
         piece.location = new_location
+        return new_location
```

Comment:
This is the `move_piece` method.

We now return the new location of the piece that was moved. This is just helpful, as it avoids callers of this class having to work it out themselves